README.md

```diff
@@ -44,6 +44,8 @@ You can then load the module in F# Interactive, and run it:
 
 More examples can be found in the [tutorial](doc/tutorial.md).
 
+👉 For auto-generators (à la AutoFixture) check out [fsharp-hedgehog-experimental](https://github.com/cmeeren/fsharp-hedgehog-experimental/).
```

Comment:
AutoFixture is so popular that I think there's no reason to add a link to it. :sweat_smile: